Failing test: test_pubsub_slow_subscriber_closed

CI: https://github.com/dragonflydb/dragonfly/actions/runs/29768559373/job/88440681887

```
        assert stats["soft_limit"] >= 1
>       assert stats["forced_disconnect"] >= 1
E       assert 0 >= 1
```

--- a/src/facade/dragonfly_connection.cc
+++ b/src/facade/dragonfly_connection.cc
@@ -1753,7 +1753,9 @@ bool Connection::ProcessControlMessages(uint32_t quota) {
       break;
     }
 
+    async_op_start_cycle_ = base::CycleClock::Now();
     std::visit(AsyncOperations{reply_builder_.get(), this}, msg.handle);
+    async_op_start_cycle_ = 0;
   }
 
   return false;

